platform/docs/docs/platform/services/data/index.md

```diff
@@ -20,6 +20,7 @@ We maintain the following non-ui Services:
 - [Toolbar Service](../data/ToolBarService.md)
 - [Measurement Service](../data/MeasurementService.md)
 - [Customization Service](customization-service.md)
+- [Panel Service](../data/PanelService.md)
```

Comment:
we need to add it to `sidebars.js` as well

Reply:
I am not sure what you mean. The deployed docs does have it in the sidebar already from what I can tell...

![image](https://user-images.githubusercontent.com/109477394/223757647-c050b092-274d-4b38-a6ad-21c5de651d2a.png)